v2/internal/pkg/release/signature.go

```diff
@@ -169,14 +170,40 @@ func (o SignatureSchema) GenerateReleaseSignatures(ctx context.Context, images [
 			}
 			img.Source = signSchema.Critical.Identity.DockerReference
 			o.Log.Debug("image found : %s", signSchema.Critical.Identity.DockerReference)
-			// o.Log.Info("public Key : %s", strings.ToUpper(fmt.Sprintf("%x", md.SignedBy.PublicKey.Fingerprint)))
 
 			// write signature to cache
 			ferr := os.WriteFile(o.Opts.Global.WorkingDir+SignatureDir+digest, data, 0644)
 			if ferr != nil {
 				o.Log.Error("%v", ferr)
 			}
 			imgs = append(imgs, img)
+			// create and store config map
+			cm := &v2alpha1.ConfigMap{
+				TypeMeta: v2alpha1.TypeMeta{
+					APIVersion: configMapApiVersion,
+					Kind:       configMapKind,
+				},
+				ObjectMeta: v2alpha1.ObjectMeta{
+					Namespace: signatureNamespace,
+					Labels: map[string]string{
+						signatureLabel: "",
+					},
+				},
+				BinaryData: make(map[string]string),
+			}
+
+			b64 := base64.RawStdEncoding.EncodeToString(data)
+			index := fmt.Sprintf(configMapBinaryDataIndexFormat, digest, id+1)
+			cm.BinaryData[index] = b64
+			jsonData, err := json.Marshal(cm)
+			if err != nil {
+				o.Log.Error("%v", err)
+			}
+			// write to cluster-resources directory
+			ferr = os.WriteFile(o.Opts.Global.WorkingDir+clusterResourcesDir+index+".json", jsonData, 0644)
+			if ferr != nil {
+				o.Log.Error("%v", ferr)
+			}
```

Comment:
Hi @lmzuccarelli,
I was wondering if this makes more sense in the `clusterresources` module, where we generate all the manifests (separation of concerns).
WDYT?